AnswerGeneratorMicroservice/helper/model downloader.py

```diff
@@ -0,0 +1,27 @@
+from transformers import pipeline
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
+from transformers import BitsAndBytesConfig
+import torch
+
+nf4_config = BitsAndBytesConfig(
+   load_in_4bit=True,
+   bnb_4bit_quant_type="nf4",
+   bnb_4bit_use_double_quant=True,
+   bnb_4bit_compute_dtype=torch.bfloat16
+)
+# Specify GPU device (replace with specific GPU index if using multiple GPUs)
+device = "cuda:0" if torch.cuda.is_available() else "cpu"
+
+# Model and tokenizer names (replace with desired model ID)
+model_name = "unsloth/OpenHermes-2.5-Mistral-7B-bnb-4bit"
```

Comment:
This also should be moved into configuration and set as a variable that its content could change